lecli/team_api.py

```diff
@@ -0,0 +1,185 @@
+import json
+
+import requests
+
+from lecli import apiutils
+
+
+def _url():
+    """
+    Get rest query url of account resource id.
+    """
+    return 'https://rest.logentries.com/management/accounts/' + str(
+        apiutils.get_account_resource_id()) + '/teams'
+
+
+def response_error(response):
+    """
+    Check response if it has any errors.
+    """
+    if response.headers.get('X-RateLimit-Remaining') is not None:
+        if int(response.headers['X-RateLimit-Remaining']) == 0:
+            print 'Error: Rate Limit Reached, will reset in ' + response.headers.get(
+                'X-RateLimit-Reset') + ' seconds'
+            return True
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as error:
+        print 'Request Error:', error.message
+        if response.status_code == 500:
+            print 'Your account may have no owner assigned. ' \
+                  'Please visit www.logentries.com for information on assigning an account owner.'
+        return True
+
+    if response.status_code == 200:
+        if response.headers['Content-Type'] != 'application/json':
+            print 'Unexpected Content Type Received in Response: ' + response.headers[
+                'Content-Type']
+            return True
+        else:
+            return False
+    return False
+
+
+def print_teams(response):
+    """
+    Print teams.
+    """
+    try:
+        print json.dumps(response.json()['teams'], indent=4, sort_keys=True)
+    except KeyError:
+        print json.dumps(response.json()['team'], indent=4, sort_keys=True)
```

Comment:
Yeah I have tried to avoid repetition here. If we query all teams, we will receive `teams` object in the response. On the other hand if we retrieve one team only, there will be `team` object in the response.